app/styles/themes/_high-contrast.scss

```
@@ -0,0 +1,68 @@
+body.theme-high-contrast {
+  // A small set of colors used to flatten ux and improve contrast
+  --hc-background-color: #090c11;
+  --hc-secondary-background-color: #323946;
+  --hc-border-color: #7a828e;
+  --hc-text-color: #f0f3f6;
+  --hc-active-item-color: #9ea7b3;
+  --hc-secondary-active-item-color: #596069;
+  --hc-active-text-color: #090c11;
+  --hc-diff-add-border-color: green;
+  --hc-diff-delete-border-color: crimson;
+
+  // background/text
+  --background-color: var(--hc-background-color);
+  --text-color: var(--hc-text-color);
+  --text-secondary-color: var(--hc-text-color);
+
+  //border
+  --hc-hover-border: 2px solid var(--hc-border-color) !important;
```

I'd rather we didn't codify specificity into the variables. And in this case I don't believe it's necessary? If it is we should move it to the specific place where it is or we'll taint all rules which include this variable with the highest specificity and then it's an `!important` war

```suggestion
  --hc-hover-border: 2px solid var(--hc-border-color);
```

---

Makes sense. Seems I remember needed it for the tab bars.. but now I can't reproduce needing it so I just removed it.